### PR TITLE
5.1 upgrade scenario

### PR DESCRIPTION
**Pull request by Ameenasuhani**
# Description
As part of 5.1 upgrades testing the I/O's have to run parallel with the upgrade

JIRA:
https://issues.redhat.com/browse/RHCEPHQE-1206
https://issues.redhat.com/browse/RHCEPHQE-1207

### CODE REVIEW

File: suites/pacific/upgrades/tier-1_upgrade_test-4x-to-5x-container.yaml [hunk 1]
```diff
@@ -0,0 +1,157 @@
+#############################################################################
```
Comment (sunilkumarn417) [inline suites/pacific/upgrades/tier-1_upgrade_test-4x-to-5x-container.yaml:1]: Hi @Ameenasuhani ,

We may need to update pipeline shell scripts with change in filenames.

**Review by sunilkumarn417 — APPROVED** [review]
LGTM

**Comment by Ameenasuhani** [comment]
let me attach the logs before merging

**Comment by Ameenasuhani** [comment]
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-4TY4ZQ/ - rbd and rgw i/o error is persistant

**Review by sunilkumarn417 — APPROVED** [review]
LGTM

**Review by psathyan — APPROVED** [review]

**Pull request merged**